src/sentry/api/urls.py

```diff
@@ -605,6 +606,21 @@
         OrganizationEventsEndpoint.as_view(),
         name='sentry-api-0-organization-events'
     ),
+    url(
+        r'^organizations/(?P<organization_slug>[^\/]+)/events/(?P<project_slug>[^\/]+):(?P<event_id>(?:\d+|[A-Fa-f0-9]{32}))/$',
```

Comment:
@markstory Is this the groupSlug value you were thinking about passing?

Reply:
I switched this to a `/` in #13568

Reply:
I'd prefer to stick with the `:`. `/` implies a subresource to me and since we can't have an eventId without a projectSlug I think it might be a bit unexpected.